cohort_4/playbook/activities.md

```diff
@@ -24,6 +29,51 @@
 
 TODO
 
+## Brainstorming
+
+### Forming groups
+
+- Assign people to groups.
+  - There are social pressures between friends when forming groups. Start by
+    assigning the hackers to random groups.
+  - If any problems arise in a group handle it on a case-by-case basis.
+
+### Yes-and sessions
+
+- In a "Yes-and" session, pairs of hackers build off each-others' ideas to create
+  long, completely nonsensical ideas
+  - "Yes-and" sessions are exercises that help people get out of a judgmental
```

Comment:
Switch the order of 43-44 with 45-46 because you want to start with why you are even doing this.

Perhaps laying down context like:
- this is problem when brainstorming
- we do this activity that has this effect to counteract it